src/components/TopNavMenu.vue

```diff
@@ -39,6 +39,10 @@ export default {
       // the next two lines are redundant because they will both happen automatically in router.js once keypair is removed.
       localStorage.removeItem('isAuthed')
       this.$router.push('/login')
+    },
+    openSettingsCloseMenu () {
```

Comment:
don't call it that :P. open close... which is it?

Reply:
does this mean that it opens settings and closes the menu? maybe just add `and`

Reply:
done 👍